source/_docs/platform-upgrade.md

```diff
@@ -0,0 +1,22 @@
+---
+title: Platform Upgrade in Progress
+description: Details on the current infrastructure maintenance.
+---
+Pantheon’s multi-tenant platform uses containers and runs on top of dedicated infrastructure. To ensure improved performance and security of this infrastructure, it periodically needs maintenance and upgrade.
+
+We at Pantheon, did not build a traditional hosting architecture. That also implies that we do not wait for our technology stack to get outdated. Thus, we constantly challenge ourselves to run our customer sites on the latest and the greatest.
+
+If your site dashboard displays **Upgrade In Progress** message, it means we are ensuring you and your customers are getting additional benefit as a result of our commitment to offer the best.

```

Comment:
If your site dashboard displays the **Platform Upgrade In Progress** message, we recommend waiting to make any code changes until maintenance completes. The maintenance window can be anywhere between 10 to 30 minutes depending on several factors including the size of your site components (code, database, files), time of the day etc.